### PR TITLE
just adds /skip as an aliases for cycle

### PR DESCRIPTION
**Pull request by DeternoMC**

### CODE REVIEW

File: TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public static void rotation(final CommandContext cmd, CommandSender sender) thro
     }
 
 
-    @Command(aliases = {"cycle"}, desc = "Cycle to a new map.")
+    @Command(aliases = {"cycle", "skip"}, desc = "Cycle to a new map.")
```
Comment (jorgeberrex) [inline TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java]: Well, the title is misleading, That's not `/next`, that's `/skip`

**Comment by lukechatton** [comment]
IMO `/next` would mean "What is the next map?".

I don't think having aliases like this are needed. Terminology should stay consistent with "cycle".

File: TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java (hunk 1 above)
Comment (DeternoMC) [reply TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java]: my mistake, apologies. Changed

**Comment by DeternoMC** [comment]
@LukeChatton, I meant /skip, just a typo on accident. I got the idea to add it from in the 'to do' list, tj added to make a /skip command where you can skip the map or an amount of maps. It was already made with /cycle, so I thought it was pointless adding a new command, it's also easier for newer moderators, or staff members that forgot the command.

However, I didn't add the part where you can specify the amount of maps to skip. I'm not an expert at coding :)

**Comment by tjhubz** [comment]
Skip is different from cycle. Cycle can only be used once in each map, while skip allows you to easily skip a map before it is loaded onto the server. I suggest looking at PGM for reference if it is not understood.

**Pull request closed without merging**